caffe2/perfkernels/embedding_lookup_idx_sve.cc

```diff
@@ -0,0 +1,6769 @@
+//// --------------------------
+//// ATTENTION:
+//// THIS CODE IS AUTOGENERATED
+//// BY sve_emblookup_codegen.py
```

Comment:
nit I would also include arguments passed to this file as well with this string, though it is just a filename now, it will help in the future.

Reply:
Done. Thanks!